setup.py

```diff
@@ -25,7 +25,7 @@
     platforms='any',
     install_requires=[
         'protobuf>=3.0.0b2.post2',
-        'opentracing==2.0.0',
+        'opentracing>=2.0.0',
```

Comment:
>=2,<3 is the preferred way

Reply:
protobuf isn't using that approach :S

Reply:
Cool I'm fine with either
Will update

Reply:
tbh not sure why this lib even needs protobuf, it might be a legacy dep